converter/fluka/input.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+from converter.fluka.card import Card
+
+
+@dataclass
+class Input:
+    """Class mapping of the fluka input file."""
+
+    energy_GeV: float = 90.0  # GeV FLUKA specific
```

Comment:
```suggestion
    energy_GeV: float = 0.15  # GeV FLUKA specific
```

This is a better default.